src/Drivers/Gd/Modifiers/TextModifier.php

```diff
@@ -29,11 +29,41 @@ public function apply(ImageInterface $image): ImageInterface
         $color = $this->driver()->colorProcessor($image->colorspace())->colorToNative(
             $this->driver()->handleInput($this->font->color())
         );
-
+        
+        $strokeWidth = $this->font->strokeWidth();
+        $strokeColor = $this->driver()->colorProcessor($image->colorspace())->colorToNative(
+            $this->driver()->handleInput($this->font->strokeColor())
+        );
```

Comment:
This color decoding process is only important if the stroke width is greater than 0 and should therefore only consume resources in this case.

Reply:
Done